spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/scutilities/HttpHandler.java

```diff
@@ -52,10 +53,14 @@ public static HttpHandler getInstance() {
     private HttpHandler() {
         this.client = new OkHttpClient.Builder()
                 .readTimeout(2, TimeUnit.MINUTES)
-                .addNetworkInterceptor(new LoggingInterceptor())
+//                .addNetworkInterceptor(new LoggingInterceptor())
```

Comment:
Do we need this? if not please remove instead of commenting it out

Reply:
It's useful for debugging when developing.  We don't need it so I can remove it...or I can try to enable it with some type of environment variable?  Let me know what you think...remove or enable it automatically in development?

Reply:
I'll remove this for now....it's not difficult to add it back if I need to debug http requests